### ISSUE TITLE
Public examples and hall of fame

### ISSUE THREAD

**Issue by dan-knight** [uclahs-cds/public-R-BoutrosLab-plotting-general#100]
Move the examples from the lab's private Confluence page into this repo (likely as discussions).

**Comment by stefaneng** [comment]
A vignette might be a better idea since then we can view it directly on the pkgdown website when anything is updated.

**Comment by pboutros** [comment]
I think we already have a vignette?

**Comment by dan-knight** [comment]
Right, we have the plotting guide vignette. It might be useful to include a section for example plots like these. However, I do think a discussion would also be useful as an easier place for users to casually contribute example plots without needing to worry about writing Rmd or making a pull request. We can then add select plots from the discussion to the vignette.

**Comment by lydiayliu** [comment]
Duplicated https://github.com/uclahs-cds/public-R-BoutrosLab-plotting-general/issues/40
